pkg/pulumiyaml/codegen/load.go

```diff
@@ -857,6 +857,13 @@ func (imp *importer) importResource(kvp ast.ResourcesMapEntry, latestPkgInfo map
 		}
 	}
 
+	if resource.Options.DeletedWith != nil {
+		resourceOptions.Body.Items = append(resourceOptions.Body.Items, &model.Attribute{
```

Comment:
@aq17, since `DeletedWith` takes a Resource, shouldn't this look more like the `if resource.Options.Parent != nil` case above? Or am I misunderstanding how this should work?

```go
	if resource.Options.DeletedWith != nil {
		ref, err := imp.getResourceRefItem(resource.Options.DeletedWith, name, "deletedWith")
		if err != nil {
			diags.Extend(err)
		} else {
			resourceOptions.Body.Items = append(resourceOptions.Body.Items, &model.Attribute{
				Name:  "deletedWith",
				Value: ref,
			})
		}
	}
```

Reply:
Yup this was incorrectly based on `DeletedWith` still accepting the URN as a string